corehq/apps/geospatial/static/geospatial/js/geospatial_map.js

```diff
@@ -487,6 +487,36 @@ hqDefine("geospatial/js/geospatial_map", [
             }
         }
 
+        // This function partitions case data into cases with and without GPS data
+        function partitionCaseData(caseData) {
+            const caseItemMapping = {
```

Comment:
This is nice. A nit and up to you if you want to change this: Since this _is_ a mapping, calling it `caseItemMapping` feels weird; `caseItem` should suffice. This is personal preference though, so totally fine to dismiss this comment

Reply:
That's a good point. This would also make the variable a little less verbose. The var name has been changed ([96045ed](https://github.com/dimagi/commcare-hq/pull/33541/commits/96045ed667c1ff9d42fa6d8d232dabbf8e8ca783)).